site/setup.cfg

```diff
@@ -30,12 +31,6 @@ invenio_assets.webpack =
     cds_rdm_theme = cds_rdm.webpack:theme
 invenio_celery.tasks =
     cds_rdm = cds_rdm.tasks
-#flask.commands =
```

Comment:
why commenting all this out, and above, and not simply removing it?

Reply:
sorry I am confused, this PR removes it, no?